spec/Records.tex

```diff
@@ -522,7 +434,7 @@ \subsection{Record Assignment}
 a default assignment operator for each record type \chpl{R} having the signature
 \begin{example}
 \begin{chapel}
-proc =(ref lhs:R, rhs:R) : void ;
+proc =(ref lhs:R, rhs) : void ;
 \end{chapel}
 \end{example}
 \noindent
```

Comment:
Below, let us specify what happens when there is a field that's in the l.h.s. and not in r.h.s. or, conversely, only in r.h.s.